data/bootstrap_seed.yaml

```diff
@@ -0,0 +1,171 @@
+---
+webvirt: 192.168.1.1
+site_name: sitename
+maas_api_key: consumerkey:tokenkey:tokensecret
+maas_uuid: nonce
+host_profiles:
+- name: k8-node
+  hardware_profile: HPGen9v3
+  interfaces:
+  # Keyed on device_name
+  # pxe is a special marker indicating which device should be used for pxe boot
+  - device_name: pxe
+    network_link: pxe
+    slaves:
+    - prim_nic01
+    networks: 
+    - name: pxe
+  - device_name: bond0
+    network_link: gp
+    slaves:
+    - prim_nic01
+    - prim_nic02
+    networks:
+    - name: mgmt
+    - name: private
+  oob:
+    type: ipmi
+    network: oob
+    account: admin
+    credential: admin
+  storage:
+    boot_drive:
+      device: primary_boot
+      bootsize: 2g
+      rootsize: 50g
+  platform:
+    image: ubuntu_16.04_hwe
+    kernel_params: default
+  metadata:
+    introspection_url: http://172.16.1.10:9090
+- name: k8-node-public
+  host_profile: k8-node
+  interfaces:
+    - device_name: bond0
+      networks:
+        # This is additive, so adds a network to those defined in the inherited host_profile
+      - name: public
+network:
+  # Describe layer 1 attributes
+  links:
+  - name: pxe
+    bonding:
+      mode: none
+    mtu: 1500
+    linkspeed: auto
+      # Is this link supporting multiple layer 2 networks?
+    trunking:
+      mode: none
+        # use name, will translate to VLAN ID
+      default_network: pxe
+  - name: gp
+    bonding:
+      mode: 802.3ad
+      hash: layer3+4
+      peer_rate: slow
+      mon_rate: default
+      up_delay: default
+      down_delay: default
+    mtu: 9000
+    linkspeed: auto
+      # Is this link supporting multiple layer 2 networks?
+    trunking:
+      mode: tagged
+  networks:
```

Comment:
My thought is the 'tagged' model can offer an attribute 'default_network' to define the native VLAN for a link. I'll fill out the example more to use the full 'schema' and add comments.